compressor_integration_tests/docker-compose.yaml

```diff
@@ -0,0 +1,16 @@
+version: '3'
+services:
+  postgres:
+    image: "postgres:latest"
+    
+    ports:
+      # N.B. format is [port on machine]:[port to expose from container]
+      - 5432:5432
+
+    environment: 
+      POSTGRES_USER: synapse_user
+      POSTGRES_PASSWORD: synapse_pass
+      POSTGRES_DB: synapse
+
```

Comment:
It may be worth setting the `PGDATA` to a tmpfs volume to make postgres go much faster

Reply:
done